svelte/components/Nft/Nft.svelte

```diff
@@ -1,250 +1,234 @@
 <script lang="ts">
   import type { Readable } from "svelte/store";
+  import { currentTokenID } from "main/current";
 
   import type { NftType } from "lib/ktypes";
   import {
-    getShortAddress,
     nftUrl,
     explorerCollectionUrl,
-    nftDescription,
-    nftDescriptionShort,
-    nftName,
-    nftOpenSeaUrl,
-    addressSame,
-    textShort,
     explorerAddressLink,
     kredeumNftUrl,
-    getNetwork
+    getNetwork,
+    nftOpenSeaUrl,
+    addressSame
   } from "lib/kconfig";
-  // import { nftKey } from "lib/kconfig";
-  import { nftGetImageLink } from "lib/knft-get-metadata";
-  import { onMount } from "svelte";
+
+  import { shortcode } from "helpers/shortcodes";
 
   import { nftStore } from "stores/nft/nft";
+
   import NftTransfer from "./NftTransfer.svelte";
 
-  /////////////////////////////////////////////////
-  //  <Nft {chainId} {address} {tokenID} {account}? {index}? {more}? {platform}? />
-  // Display NFT
-  /////////////////////////////////////////////////
   export let chainId: number;
   export let address: string;
   export let tokenID: string;
   export let account: string = undefined;
-  export let index = 0;
-  export let more = 0;
   export let platform: string = "dapp";
-
   let nft: Readable<NftType>;
 
   // let i = 1;
   // HANDLE CHANGE : on truthy chainId and address, and whatever account
   $: account, chainId && address && tokenID && handleChange();
   const handleChange = (): void => {
-    // console.log(`NFT CHANGE #${i++} ${nftKey(chainId, address, tokenID)}`);
+    // console.log(`NFTDETAIL CHANGE #${i++} ${nftKey(chainId, address, tokenID)}`);
 
     // STATE VIEW : sync get Nft
     nft = nftStore.getOneStore(chainId, address, tokenID);
 
     // ACTION : async refresh Nft
-    nftStore.refreshOne(chainId, address, tokenID).catch(console.error);
-  };
-
-  const moreToggle = (): void => {
-    more = more > 0 ? 0 : (document.getElementById(`more-detail-${index}`)?.offsetHeight || 0) + 70;
+    nftStore.refreshOne(chainId, address, tokenID);
```

Comment:
39:5  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises

use .catch(console.error) like in deleted line 50